src/Pages/ProductList/Components/Aside/Aside.js

```diff
@@ -1,13 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './Aside.scss';
 
 const Aside = () => {
+  const [clickedList, setIsClickedList] = useState('0');
+  const filterList = [
+    {
+      name: 'product Type',
+    },
+    {
+      name: 'Size',
+    },
+    {
+      name: 'Character',
+    },
+  ];
```

Comment:
여기서 id 값도 주어서 밑 map 에서 id를 이용해주세요:)
그리고 이부분은 상수데이터에 가까울 것 같네요. 
이 값 같은 경우에는 매 rendering 마다 새롭게 선언되고 할당될 필요는 없어보이기 때문에 export default Aside; 밑에 선언
상수데이터의 네이밍은 상수데이터임을 표시하기 위해서 UPPERCASER + snake_case 로 작성해주는게 컨벤션입니다

```suggestion
  const filterList = [
    {
      id:
      name: 'product Type',
    },
    {
      id:
      name: 'Size',
    },
    {
      id:
      name: 'Character',
    },
  ];
```

Reply:
네 수정하였습니다. 감사합니다!